lbz/handlers.py

```diff
@@ -9,11 +9,16 @@
 
 
 class BaseHandler(Generic[T], metaclass=ABCMeta):
+    def __init__(self, event: dict, context: object = None) -> None:
```

Comment:
Now, when I look at this I think `context` should not be optional 🤔 
```suggestion
    def __init__(self, event: dict, context: object) -> None:
```

Reply:
If we keep it like this, we will have to add an `if` condition every time when using this value 😐